Core/src/org/sleuthkit/autopsy/modules/filetypeid/FileType.java

```diff
@@ -102,6 +103,31 @@ boolean alertOnMatch() {
     String getFilesSetName() {
         return interestingFilesSetName;
     }
+    
+    @Override
+    public String toString() {
+        return this.mimeType;
+    }
+    
+    @Override
+    public boolean equals(Object other) {
+        if(other != null && other instanceof FileType) {
```

Comment:
I think this equals/hashcode pair is broken.  Two filetypes that are equal based on their comparison of mimetype and signature could have different hashcodes becuase the hash code also includes the interesting file set name and alert boolean.